Doc/library/typing.rst

```diff
@@ -1536,8 +1703,17 @@ These are not used in annotations. They are building blocks for creating generic
    :class:`TypeVar`.  However the actual semantics of these keywords are yet to
    be decided.
 
+   .. attribute:: __name__
+
+      The name of the parameter specification.
+
```

Comment:
I think this should be moved above the preceding paragraph, so that it comes directly after the reference for the `args` and `kwargs` attributes